openfecwebapp/templates/partials/independent-expenditures-filter.html

```diff
@@ -20,7 +20,7 @@
 <div class="js-accordion accordion--neutral" data-content-prefix="filter" data-open-first="false">
   <div class="filters__inner">
     {{ typeahead.field('committee_id', 'Spender name or ID') }}
-    {{ years.years('cycle', 'Two-year period') }}
+    {{ years.years('cycle', 'Years active') }}
```

Comment:
Again, "years active" makes me think years that the group is active, in this case we are looking at all events of a particular type in a two-year time span. Can we just say "years" or something like that?